modules/statistics/templates/progress_bar.tpl

```diff
@@ -1,6 +1,6 @@
   {if $project['recruitment_target'] neq ""}
     <h5>{$project['title']}</h5>
-    {if $project['surpassed_recruitment'] eq "true"}
+    {if isset($project['surpassed_recruitment']) && $project['surpassed_recruitment'] eq "true"}
```

Comment:
Notice: Updated smarty requires isset check for variables. There might be a way to turn this off but it gets rid of the warnings for now when just checking isset

Reply:
I'm pretty sure there's a setting that we had to change with a previous smarty release (though I can't remember what it is..)

Reply:
@driusan I spent a few hours trying to get `$smarty->error_reporting = E_ALL & ~E_NOTICE;` to work and gave up. Turns out `$whatever_variable|default` mimics the previous behaviour of setting it to an empty string if undefined.